spec/features/courses/vacancies/edit_spec.rb

```diff
@@ -0,0 +1,243 @@
+require 'rails_helper'
+
+feature 'Edit course vacancies', type: :feature do
+  let(:base_course) do
+    {
+      data: {
+        id: '1',
+        type: 'courses',
+        attributes: {
+          has_vacancies?: true,
+          course_code: 'C1D3',
+          name: 'English',
+          study_mode: 'full_time_or_part_time'
+        },
+        relationships: {
+          provider: {
+            meta: {
+              included: false
+            }
+          },
+          site_statuses: {
+            data: [
+              {
+                type: 'site_statuses',
+                id: '1'
+              },
+              {
+                type: 'site_statuses',
+                id: '2'
+              },
+              {
+                type: 'site_statuses',
+                id: '3'
+              },
+              {
+                type: 'site_statuses',
+                id: '4'
+              }
+            ]
+          }
+        }
+      },
+      included: [
+        {
+          id: '1',
+          type: 'site_statuses',
+          attributes: {
+            vac_status: 'both_full_time_and_part_time_vacancies'
+          },
+          relationships: {
+            site: {
+              data: {
+                type: 'sites',
+                id: '3'
+              }
+            }
+          }
+        },
+        {
+          id: '2',
+          type: 'site_statuses',
+          attributes: {
+            vac_status: 'full_time_vacancies'
+          },
+          relationships: {
+            site: {
+              data: {
+                type: 'sites',
+                id: '3'
+              }
+            }
+          }
+        },
+        {
+          id: '3',
+          type: 'site_statuses',
+          attributes: {
+            vac_status: 'part_time_vacancies'
+          },
+          relationships: {
+            site: {
+              data: {
+                type: 'sites',
+                id: '3'
+              }
+            }
+          }
+        },
+        {
+          id: '4',
+          type: 'site_statuses',
+          attributes: {
+            vac_status: 'no_vacancies'
+          },
+          relationships: {
+            site: {
+              data: {
+                type: 'sites',
+                id: '3'
+              }
```

Comment:
All the `SiteStatus` objects here have the same `Site`. That doesn't quite sound right to me, is it? If not, should we bother to make this better represent what's actually real data?

In a future refactor maybe?